src/ElmFormat/Execute.hs

```diff
@@ -1,7 +1,7 @@
 {-# LANGUAGE Rank2Types #-}
 module ElmFormat.Execute (forHuman, forMachine, run) where
 
-{-| This module provides executors that can take streams of Operations and
+{- This module provides executors that can take streams of Operations and
```

Comment:
I had to do this change because GHC 8.8.1 was complaining about parsing error. Seems this style of comment is now possible only above function definition not above imports.